Use the variables defined by the KDE4 cmake files to link against kdecorations.

--- client/CMakeLists.txt.orig	2017-03-29 09:58:23 UTC
+++ client/CMakeLists.txt
@@ -77,7 +77,7 @@ set(kwin3_crystal_PART_SRCS
    )
 
 kde4_add_plugin(kwin3_crystal ${kwin3_crystal_PART_SRCS})
-target_link_libraries(kwin3_crystal ${KDE4_KDEUI_LIBS} kdecorations )
+target_link_libraries(kwin3_crystal ${KDE4_KDEUI_LIBS} ${KDE4WORKSPACE_KDECORATIONS_LIBS} )
 
 install(TARGETS kwin3_crystal  DESTINATION ${PLUGIN_INSTALL_DIR} )
 install( FILES crystal.desktop  DESTINATION  ${DATA_INSTALL_DIR}/kwin/ )
